editor/src/components/editor/store/editor-state.ts

```diff
@@ -1040,10 +1047,16 @@ export function mergeStoredEditorStateIntoEditorState(
   if (storedEditorState == null) {
     return editor
   } else {
+    // for unknown mode types, like select-lite
```

Comment:
maybe nitpicky, but I think this would be better as a migration step!

Reply:
I reverted this and added `version` to StoredState instead. `loadStoredState` won't load the stored state if the version is not matching to the latest storedstate version causing it to reset.